feat: Better async handling

joostvanderborg opened the pull request:
- retries calls (3 times) if the api is too slow
- removes 'show zaken' from env, no longer needed.
- adds option to show taken (on homepage + on zaak page)
- fixes timeouts (reversed)

As well as:
- NL Wallet new connections (feature flagged)
- Login mechanism in time generation of auth redirect links

joostvanderborg committed:
feat: Add alle zaken-link to homepage

joostvanderborg committed:
feat: Add taken to homepage

joostvanderborg committed:
chore: refactor, remove config for showing zaken

joostvanderborg committed:
fix: Use correct template

joostvanderborg committed:
fix: Send 'taken' to template

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11135623701

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1268 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

joostvanderborg committed:
feat: Add taken styling

joostvanderborg committed:
Merge branch 'acceptance' into development

joostvanderborg committed:
Merge pull request #1269 from GemeenteNijmegen/development

feat: Add taken to home

joostvanderborg committed:
Merge branch 'development' of github.com:GemeenteNijmegen/mijn-nijmegen into development

joostvanderborg committed:
feat: fix timeouts, add better retries

joostvanderborg committed:
fix: fix test

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11153679096

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1270 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

joostvanderborg committed:
fix: don't throw error before loading

joostvanderborg committed:
Merge branch 'development' of github.com:GemeenteNijmegen/mijn-nijmegen into development

joostvanderborg committed:
feat: Retry handling of json responses

If the data can't load on the frontend, we will retry for a total of 4
times, trying to load the data anyway.

joostvanderborg committed:
fix test

joostvanderborg committed:
Merge branch 'acceptance' into development

joostvanderborg committed:
Merge pull request #1274 from GemeenteNijmegen/development

feat: Better async handling

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11171799698

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1276 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11188796649

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1277 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11197458260

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1278 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11206469200

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1279 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11226237012

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1280 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11245902477

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1281 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11265161729

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1282 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

joostvanderborg committed:
fix: Show closed cases

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11284079208

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1283 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11301038700

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1284 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11309915320

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1285 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11318963550

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1286 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11337024276

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1291 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

marnixdessing committed:
feat: setup login lambda for NL Wallet beproeving

marnixdessing committed:
chore: diplicate methods in preparation of different configurations

marnixdessing committed:
feat: introduce alternative OpenIDConnectV2 class

marnixdessing committed:
chore: test OpenIDConnectV2 class

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11356547704

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1292 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

joostvanderborg committed:
feat: Update task template

marnixdessing committed:
feat: add configuration env vars for verid nlwallet

joostvanderborg committed:
feat: Update task template

marnixdessing committed:
Merge pull request #1293 from GemeenteNijmegen/feat/nl-wallet-logins

feat: nl-wallet login verId

marnixdessing committed:
feat: add feature flag for nl wallet

marnixdessing committed:
Merge pull request #1294 from GemeenteNijmegen/feat/nl-wallet-logins

feat: add feature flag for nl wallet

marnixdessing committed:
chore: make sure nl wallet login urls are valid

marnixdessing committed:
chore: remove additional / after base url

marnixdessing committed:
chore: split feature flag for nl wallet between two brokers

marnixdessing committed:
chore: pass feature flag settings to auth handler

marnixdessing committed:
chore: add authorization url creation for signicat

marnixdessing committed:
chore: add signicat nl wallet auth flow in handler

marnixdessing committed:
chore: correct different state

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11376025911

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1295 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

marnixdessing committed:
chore: add OIDC debug logging

marnixdessing committed:
Merge branch 'development' of github.com:GemeenteNijmegen/mijn-nijmegen into development

marnixdessing committed:
chore: add client options to OpenIDConnectV2

marnixdessing committed:
chore: add parameters for nl wallet signicat

marnixdessing committed:
chore: pass signicat configuration to login and auth lambda

marnixdessing committed:
feat: enable signicat nl wallet flow in dev

marnixdessing committed:
chore: fix tests after adding parameters and secrets

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11395168471

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1296 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

marnixdessing committed:
chore: allow auth lambda to get client secret signicat

marnixdessing committed:
Merge branch 'development' of github.com:GemeenteNijmegen/mijn-nijmegen into development

marnixdessing committed:
feat: just in time generation of login links (#1297)

* feat: login lambda with just in time redirect url building

* feat: make auth lambda use auth method in state

* chore: update unit tests after updating login page

marnixdessing committed:
chore: make sure nl wallet login urls can be build as well

marnixdessing committed:
feat: NL Wallet preparations (#1298)

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11171799698

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11188796649

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11197458260

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11206469200

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11226237012

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11245902477

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11265161729

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* fix: Show closed cases

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11284079208

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11301038700

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11309915320

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11318963550

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11337024276

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* feat: setup login lambda for NL Wallet beproeving

* chore: diplicate methods in preparation of different configurations

* feat: introduce alternative OpenIDConnectV2 class

* chore: test OpenIDConnectV2 class

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11356547704

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* feat: Update task template

* feat: add configuration env vars for verid nlwallet

* feat: Update task template

* feat: add feature flag for nl wallet

* chore: make sure nl wallet login urls are valid

* chore: remove additional / after base url

* chore: split feature flag for nl wallet between two brokers

* chore: pass feature flag settings to auth handler

* chore: add authorization url creation for signicat

* chore: add signicat nl wallet auth flow in handler

* chore: correct different state

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11376025911

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* chore: add OIDC debug logging

* chore: add client options to OpenIDConnectV2

* chore: add parameters for nl wallet signicat

* chore: pass signicat configuration to login and auth lambda

* feat: enable signicat nl wallet flow in dev

* chore: fix tests after adding parameters and secrets

* chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11395168471

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

* chore: allow auth lambda to get client secret signicat

* feat: just in time generation of login links (#1297)

* feat: login lambda with just in time redirect url building

* feat: make auth lambda use auth method in state

* chore: update unit tests after updating login page

* chore: make sure nl wallet login urls can be build as well

---------

Signed-off-by: github-actions <github-actions@github.com>
Co-authored-by: github-actions <github-actions@github.com>
Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
Co-authored-by: Joost van der Borg <j.van.der.borg@nijmegen.nl>

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11412861591

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1299 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11421908451

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1300 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11450888214

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1309 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11470699123

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1312 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11490307067

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1313 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

joostvanderborg committed:
timeout settings

joostvanderborg committed:
Merge branch 'development' of github.com:GemeenteNijmegen/mijn-nijmegen into development

joostvanderborg committed:
feat: extract taken template

joostvanderborg committed:
feat: Dynamically load zaak

joostvanderborg committed:
fix: await render, add xsrf

joostvanderborg committed:
feat: Better loading indicator

joostvanderborg committed:
Merge branch 'acceptance' into development

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11509599562

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1315 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11527247276

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1316 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11536099684

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1317 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11545121391

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1318 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11564947680

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1319 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

marnixdessing committed:
fix: remove resource url from OIDCv2 impl

marnixdessing committed:
chore: add login prompt to auth url oidcv2

marnixdessing committed:
Revert "chore: add login prompt to auth url oidcv2"

This reverts commit c13c8cf361671a914eecf1fba74b07d316375206.

marnixdessing committed:
feat: add user info endpoint support in OIDCv2

marnixdessing committed:
chore: add addiontal logging

marnixdessing committed:
chore: simplify callback construction

joostvanderborg committed:
Merge pull request #1314 from GemeenteNijmegen/development

feat: release

marnixdessing committed:
chore: log error as we cannot always format it

marnixdessing committed:
chore: custom callback using fetch to test

marnixdessing committed:
chore: add issue in token callback

marnixdessing committed:
chore: refresh_token not alway available (in case of nl wallet)

WietekeNijmegen commented on src/app/home/homeRequestHandler.ts:
```diff
@@ -4,30 +4,31 @@ import { Session } from '@gemeentenijmegen/session';
 
 interface HomeRequestHandlerProps {
   /**
    * Show zaken in navigation
```
Als je toch bezig bent. De comment klopt niet helemaal meer. Niet heel relevant

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11584672669

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1320 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

invalid-email-address committed:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/11603827840

------

*Automatically created by projen via the "upgrade-development" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #1321 from GemeenteNijmegen/github-actions/upgrade-development

chore(deps): upgrade dependencies

joostvanderborg committed:
fix: statusses for zaken'

joostvanderborg committed:
Merge branch 'acceptance' into development

joostvanderborg committed:
Merge pull request #1324 from GemeenteNijmegen/development

fix: fixes for taken, wallet

marnixdessing reviewed (approved):
Heel veel functionaliteit met het nieuwe taken patroon. Bevat aanpassingen aan de front-end (incl. home page). Getest met 999999333 op accp. Ziet er goed uit. Wanner zaken of taken niet optijd zijn geladen worden deze niet weergegeven op de overzichtspagina.
